### ISSUE TITLE
create haskell data from .proto message files (and vice versa)

### ISSUE THREAD

**Issue by ghorn** [alphaHeavy/protobuf#9]
The documentation mentions this is in the works already. It would be nice to have this, especially since the hprotoc project seems to be broken on ghc 7.8

**Comment by YellowOnion** [comment]
In the meantime, is there anyway to manually convert .proto to work with this library.

**Comment by NathanHowell** [comment]
@YellowOnion yes, you can manually translate messages in a .proto file into their protobuf equivalent records in Haskell... for example most of the records in https://github.com/alphaHeavy/protobuf/blob/master/plugin/Main.hs were translated from https://github.com/google/protobuf/blob/master/src/google/protobuf/descriptor.proto
